src/content/reference/runtime-apis/response.md

```diff
@@ -35,20 +35,29 @@ Valid options for the `options` object include:
   - `statusText` <Type>string</Type>
     - The status message associated with the status code, like, `OK`.
 
-  - `headers` <TypeLink href="/reference/apis/request#constructor-parameters">Headers</TypeLink> | <TypeLink href="https://developer.mozilla.org/en-US/docs/Web/API/ByteString">ByteString</TypeLink>
-    - Any headers to add to your response that are contained within a [`Headers`](/reference/apis/request#constructor-parameters) object or object literal of [`ByteString`](https://developer.mozilla.org/en-US/docs/Web/API/ByteString) key/value pairs.
+<<<<<<< HEAD
```

Comment:
@davidtsong you have a couple of merge conflicts in this file; can you fix?

Reply:
gotcha, forgot to save the file haha